pypsa/linopf.py

```diff
@@ -1035,9 +1035,11 @@ def save_optimal_capacities(n, iteration, status):
     logger.info('Running last lopf with fixed branches, overwrite p_nom '
                 'for links and s_nom for lines')
     ext_links_i = get_extendable_i(n, 'Link')
+    original_lnv = n.lines.s_nom.copy()
+    original_lkv = n.links.p_nom.copy()
```

Comment:
maybe rename `original_lnv` to `s_nom_orig`? This would fit better with the attribute name.